include/scrnintstr.h

```diff
@@ -709,12 +709,12 @@ typedef struct _ScreenInfo {
     int bitmapScanlineUnit;
     int bitmapScanlinePad;
     int bitmapBitOrder;
-    int numPixmapFormats;
-     PixmapFormatRec formats[MAXFORMATS];
-    int numScreens;
     ScreenPtr screens[MAXSCREENS];
-    int numGPUScreens;
     ScreenPtr gpuscreens[MAXGPUSCREENS];
+    PixmapFormatRec formats[MAXFORMATS];
```

Comment:
we can't change it - breaks ABI - and most likely also used by proprietary NVidia driver